src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -1,901 +0,0 @@
-<template>
-	<www-page class="lend-by-category-page">
-		<lend-header :filter-url="leadHeaderFilterLink" :side-arrows-padding="true" />
-
-		<featured-hero-loan-wrapper
-			v-if="showFeaturedHeroLoan"
-			ref="featured"
-			:is-logged-in="isLoggedIn"
-			:items-in-basket="itemsInBasket"
-			:show-category-description="showCategoryDescription"
-			:use-category-service="categoryServiceExpActive"
-			:enable-five-dollars-notes="enableFiveDollarsNotes"
-			@loaded="trackFeaturedLoan"
-		/>
-
-		<div class="tw-bg-primary">
-			<div
-				class="loan-category-row"
-				:class="{'hover-row': showHoverLoanCards}"
-				v-for="(category, index) in categories"
-				:key="category.id"
-			>
-				<component
-					:is="categoryRowType"
-					:loan-channel="category"
-					:items-in-basket="itemsInBasket"
-					:row-number="index + 1"
-					:set-id="categorySetId"
-					:is-logged-in="isLoggedIn"
-					:show-category-description="showCategoryDescription"
-					:show-expandable-loan-cards="false"
-					:enable-five-dollars-notes="enableFiveDollarsNotes"
-					ref="categoryRow"
-					@add-to-basket="handleAddToBasket"
-				/>
-			</div>
-		</div>
-
-		<div class="row pre-footer" ref="bottom">
-			<div class="column small-12">
-				<div v-if="!rowLazyLoadComplete" class="cat-row-loader">
-					<kv-loading-overlay id="updating-overlay" />
-					<h3 class="tw-text-center">
-						Loading more rows...
-					</h3>
-				</div>
-
-				<h2 class="category-name">
-					<router-link
-						:to="{ path: '/categories'}"
-						class="view-all-link"
-					>
-						View all categories<span class="view-all-arrow">&rsaquo;</span>
-					</router-link>
-				</h2>
-			</div>
-		</div>
-
-		<add-to-basket-interstitial />
-
-		<m-g-lightbox />
-		<m-g-digest-lightbox v-if="showMGDigestLightbox" />
-	</www-page>
-</template>
-
-<script>
-import _each from 'lodash/each';
-import _filter from 'lodash/filter';
-import _get from 'lodash/get';
-import _map from 'lodash/map';
-import _uniqBy from 'lodash/uniqBy';
-import _without from 'lodash/without';
-import { addCustomChannelInfo } from '@/util/categoryUtils';
-import logReadQueryError from '@/util/logReadQueryError';
-import { readJSONSetting } from '@/util/settingsUtils';
-import { indexIn } from '@/util/comparators';
-import { FLSS_ORIGIN_LEND_BY_CATEGORY } from '@/util/flssUtils';
```

Comment:
Please ask Rob if we should be using this `FLSS_ORIGIN_LEND_BY_CATEGORY` on the new lending home page now that it lives at `/lend-by-category`, or if we should continue to use `FLSS_ORIGIN_LENDING_HOME`. We want to continue using `FLSS_ORIGIN_LENDING_HOME`, then `FLSS_ORIGIN_LEND_BY_CATEGORY` can be removed from `flssUtills`. My guess is that we want to use `FLSS_ORIGIN_LEND_BY_CATEGORY` in the new page, but we'll want to verify first.

Reply:
I saw the commit to change the origin. Is that what the data team prefers?

Reply:
Yeah, confirmation is in the ticket.